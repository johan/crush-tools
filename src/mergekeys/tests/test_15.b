Site	Ad Type	Month	Activity-Type	Ad Size
Page 3	Standard	June	abc123	468x60
Page 3	Standard	June	xzy987	468x60
Page 3	Time Based Bulk	June	abc123	Not Selected
Page 3	Time Based Bulk	June	xyz987	Not Selected
Page 3	Time Based Bulk	June	deadbeef	Not Selected
The	Exclusive	June	abc123	125x125
The	Exclusive	June	deadbeef	125x125
