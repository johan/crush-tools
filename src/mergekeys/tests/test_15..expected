Month	Site	Ad Size	Ad Type	Impressions	Clicks	Activity-Type
June	Page 3	468x60	Standard	1008405	9	abc123
June	Page 3	468x60	Standard	1008405	9	xzy987
June	Page 3	Not Selected	Standard	160376	527	
June	Page 3	Not Selected	Time Based Bulk			abc123
June	Page 3	Not Selected	Time Based Bulk			xyz987
June	Page 3	Not Selected	Time Based Bulk			deadbeef
June	Sales	728x90	Exclusive	3229	30	
June	The	125x125	Exclusive	1042	25	abc123
June	The	125x125	Exclusive	1042	25	deadbeef
